video/out/opengl/context_drm_egl.c

```diff
@@ -708,6 +708,11 @@ static bool drm_egl_reconfig(struct ra_ctx *ctx)
     return true;
 }
 
+static bool drm_egl_pass_colorspace(struct ra_ctx *ctx)
+{
+    return true;
```

Comment:
It is fine for now, but I would expect it to be smarter and actually return true only if we are actually able to signal colorspace and is supported by display. Also this could instead of bool return supported modes.

Reply:
I thought about doing that but drm already handles that on its own anyway so it seemed unnecessarily smart.

Reply:
If you would return `false` from here, gpu-next would nicely fallback to sRGB. By returning always `true` it will always try to output HDR PQ, even if not wanted.

Reply:
Added a little something which I think should work.